libs/output-mapping/tests/Writer/Table/Source/AbsWorkspaceItemSourceFactoryTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Tests\Writer\Table\Source;
+
+use Keboola\FileStorage\Abs\ClientFactory;
+use Keboola\InputMapping\Staging\ProviderInterface;
+use Keboola\OutputMapping\Exception\InvalidOutputException;
+use Keboola\OutputMapping\Tests\AbstractTestCase;
+use Keboola\OutputMapping\Tests\InitSynapseStorageClientTrait;
+use Keboola\OutputMapping\Writer\Table\Source\AbsWorkspaceItemSourceFactory;
+use Keboola\StorageApi\ABSUploader;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Temp\Temp;
+use MicrosoftAzure\Storage\Blob\Models\CreateBlockBlobOptions;
+
+class AbsWorkspaceItemSourceFactoryTest extends AbstractTestCase
+{
+    use InitSynapseStorageClientTrait;
+
+    protected function initClient(?string $branchId = ''): void
+    {
+        $this->clientWrapper = $this->getSynapseClientWrapper();
+    }
+
+    public function setUp(): void
+    {
+        if (!$this->checkSynapseTests()) {
+            self::markTestSkipped('Synapse tests disabled.');
+        }
+        parent::setUp();
+    }
+
+    public function testCreateSource(): void
+    {
+        $workspaces = new Workspaces($this->clientWrapper->getBranchClient());
+        $workspace = $workspaces->createWorkspace(['backend' => 'abs'], true);
```

Comment:
ten worksapce by to mohlo dropnout

Reply:
Ten se přece smaže sám tu https://github.com/keboola/platform-libraries/blob/3e25ffc6f587561abc57a9408093131b6e8d5d3a/libs/output-mapping/tests/AbstractTestCase.php#L104